analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -437,6 +431,14 @@ class Pip(
         return declaredLicenses
     }
 
+    private fun getLicenseFromLicenseField(value: String?): String? =
```

Comment:
The fact that a "license" field is used is actually something that's implemented in the caller and this function does not know about. Hence, I'd probably just call this "getLicenseFromString".

Reply:
Because the function is named `getLicenseFromLicenseField` it is implied that the license field value has to be passed. So the function does know about it. I prefer to keep it `as-is` also because `getLicenseFromString` is harder to distinguish from `getLicenseFromClassifier`. Are you fine to keep it as is?